app/assets/stylesheets/application.css.scss

```diff
@@ -12,4 +12,45 @@
  *
  *= require_tree .
  *= require_self
+ *= require foundation_and_overrides
+
  */
+
+html body {
```

Comment:
Generally we try not to put CSS in this particular file and instead put this sort of stuff in a `_base.scss` or `_layout.scss` file that gets required first (or close to first).